### PR TITLE
Switch to `solvuu-build`

### PR DESCRIPTION
**Pull request by smondet**
Then I think I can deprecate https://github.com/smondet/omake-hacks :tada:

### CODE REVIEW

File: opam/opam [hunk 1]
```diff
@@ -5,17 +5,16 @@ maintainer: "seb@mondet.org"
 authors: [ "Sebastien Mondet <seb@mondet.org>" ]
 license: "Apache-2.0"
 build: [
-  ["omake"]
-  ["omake" "install" "BINDIR=%{bin}%"]
-]
-remove: [
```
Comment (ihodes) [inline opam/opam:11]: Why don't we need this anymore?

Comment (smondet) [reply opam/opam:11]: All solvuuu-build-based `opam` files use the `<project>.install` format to tell opam how to install/uninstall, so no need for a `remove` entry

Comment (ihodes) [reply opam/opam:11]: Cool!

**Review by ihodes — APPROVED** [review]
LGTM! Glad to be on solvuu-build now.

**Pull request merged**